.github/workflows/build_docs.yml

```diff
@@ -14,7 +14,7 @@ on:
 jobs:
   CXX:
     container:
-      image: vowpalwabbit/ubuntu1604-build:0.6.0
+      image: vowpalwabbit/ubuntu1604-build:latest
```

Comment:
Prior to merging these will need to be moved to a tagged version

Reply:
We were actually using different versions in different builds (i.e. some images were `0.6.0`, others `0.5.0` etc). I think it's nice to pick up latest in our CI builds just so we stay up to date without having to update the builds/forget to update some of them, and pickup braking changes early. We could still create another tag that includes the flatbuffers change.

Reply:
The problem is when a new build accidentally breaks builds - locking on a tagged build means we selectively move forward when ready. But you're right, on the other hand being latest is nice and knowing about the failures sooner is helpful too.